tdmq/query_builder.py

```diff
@@ -1,21 +1,22 @@
-import json
 import datetime
+import json
+
 import psycopg2.sql as sql
 from psycopg2.extras import Json
 
 
-def select_sensors(args):
+def select_sensors_by_footprint(args):
     assert args['footprint']['type'] in ['circle']
     if args['footprint']['type'] == 'circle':
         return select_sensors_in_circle(args)
 
 
-def select_sensor_types(args):
+def filter_by_description(table_name, args):
     """\
     E.g., dict(brandName="Acme", controlledProperty="humidity,temperature")
     """
     query = "".join([
-        "SELECT description FROM sensor_types WHERE",
+        "SELECT description FROM %s WHERE" % table_name,
```

Comment:
Now that the table name is variable we need to build the query with psycopg2's `sql.SQL` objects. I will open a PR on your fork, it's the easiest path.